tests/suites/test_suite_pkcs7.function

```diff
@@ -0,0 +1,561 @@
+/* BEGIN_HEADER */
+#include "mbedtls/bignum.h"
+#include "mbedtls/pkcs7.h"
+#include "mbedtls/x509.h"
+#include "mbedtls/x509_crt.h"
+#include "mbedtls/x509_crl.h"
+#include "mbedtls/oid.h"
+#include "sys/types.h"
+#include "sys/stat.h"
+/* END_HEADER */
+
+/* BEGIN_DEPENDENCIES
+ * depends_on:MBEDTLS_PKCS7_C
+ * END_DEPENDENCIES
+ */
+
+/* BEGIN_CASE depends_on:MBEDTLS_FS_IO:MBEDTLS_RSA_C */
+void pkcs7_parse( char *pkcs7_file )
```

Comment:
Please unify these test cases - I think many (all?) of them could be combined into a single test case with some additional parameters to specify e.g. expected return value from `mbedtls_pkcs7_parse_der`.

Reply:
Okay, I agree. I would like to see my most recent update pass CI first and then I will come back to fix test suite functions.